internal_transfers/actions/tests/simulate-tenderly.spec.ts

```diff
@@ -0,0 +1,61 @@
+      "0",
+      false,
+      16300366
+    );
+    expect(simulation.transaction).toMatchSnapshot();
```

Comment:
what is the goal of this TEST? 

Is to test that tenderly don't change their API? 

Just for calling a simulation  with a call-data and checking the result you could have a script (if this is what you want to test). 

Unit testing would be more for verifying the implementation of your components in isolation, so you wold need to mock the result and set some expectation on what you want your code to do.

Reply:
Actually, the goal of this (skipped) test just for devs to be able to check that the `simulate` function works as implemented. It turns out (as you mentioned that the error handling does not belong where I put it, so perhaps this does not make sense anymore. I guess the test should really just `expect(response.status).toEqual(200)`.